grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -725,15 +726,31 @@ class QueryService implements Status {
 
         def rows = getSqlConnection().rows(query, params)
         def ids = rows.collect{it.get("id")}
-        Collection result = Collections.emptyList()
+        Set result = new HashSet<Issue>()
 
         if (ids.size() > 0) {
-            result = Issue.findAllByIdInList(ids)
+            result = findIssuesSearchItemsDTO(ids)
         }
         result
     }
 
-
+    /**
+     * Find issues from a list of Ids
+     * @param issueIds
+     * @return
+     */
+    Set<Issue> findIssuesSearchItemsDTO(ArrayList<Integer> issueIds) {
+        final String query = "SELECT * FROM issue i WHERE i.id IN (:issueIds)"
+        SessionFactory sessionFactory = grailsApplication.getMainContext().getBean('sessionFactory')
+        final session = sessionFactory.currentSession
+        final SQLQuery sqlQuery = session.createSQLQuery(query)
+        final results = sqlQuery.with {
+            addEntity(Issue)
+            setParameterList('issueIds', issueIds)
+            list()
+        }
+        results
```

Comment:
@tadeork - you are correct here, this will be a little slower than processing the raw `GroovyRowResults` directly. You don't *have* to use hibernate here (you are with the `addEntity` call). I don't know the exact syntax off-hand, but it might be something as simple as 
```
        final results = sqlQuery.with {
            setParameterList('issueIds', issueIds)
            list()
        }
```
And that should give you a list of raw row results without having to cast each one as an `Issue` object (which happens under the hood by GORM/hibernate).

Reply:
This is not working. I need some how to cast the results to Issues.

Reply:
Something like this should also work - it doesn't use hibernate at all (sqlQuery still uses hibernate to generate a raw sql query ... fast, but not as fast as groovy sql)

```
getSqlConnection().rows(query, ["issueIds": issueIds]).collect { it -> ... make something from the it, which is a GroovyRowResult ... }
```

Reply:
I already tried this approach and also didn't work. It doesn't returns any result. I tried several approaches and it was frustrating bcs only a few ones work but slowly.

Reply:
Yeah, I was running a local instance to test and it does look like that doesn't work. No worries on this topic then.